public/includes/import-export.php

```diff
@@ -88,10 +88,15 @@ public function get_export_json( $ids=array() ) {
 	public function admin_menu() {
 		global $submenu;
 
-		add_submenu_page( 'edit.php?post_type=da_image', __( 'Import / Export', 'draw-attention' ), __( 'Import / Export', 'draw-attention' ), 'edit_posts', 'import_export', array( $this, 'output_import_export_page' ) );
+		add_submenu_page( 'edit.php?post_type=da_image', __( 'Import / Export', 'draw-attention' ), __( 'Import / Export', 'draw-attention' ), 'delete_others_posts', 'import_export', array( $this, 'output_import_export_page' ) );
 	}
 
 	public function output_import_export_page() {
+		// only allow admins and editors
+		$user = wp_get_current_user();
+		if( ! array_intersect( array('administrator', 'editor' ), (array) $user->roles ) ) {
```

Comment:
@hsein-bitar I believe we want to bind to the `capability` here rather than the `role`. As written, only the default WP roles would work, but if you make a new custom role (using a plugin like Members) like `Content Manager` that has the ability to `delete_others_posts`, then it should still let you see this page. Another case would be if they modify the default Editor role so that it doesn't have permission to delete others' posts for some reason. For these cases, we want to test the capability rather than the role

I'd just make this function call `if ( current_user_can( 'delete_others_posts' ) )` so that it tests for the actual capability we want rather than explicitly checking the role

Reply:
@tylerdigital thanks for the tip